.github/workflows/generate.yml

```diff
@@ -56,7 +56,32 @@ jobs:
         with:
           version: "v0.7.0"
 
+      - name: Upgrade-Tests
```

Comment:
This file will be reverted back before merging. These changes are to make sure all the tests in CI are passing as this PR involves replace helm CLI with helm library.